src/utils/spid.ts

```diff
@@ -81,46 +81,46 @@ type SpidMsg = t.TypeOf<typeof SpidMsg>;
 
 export const makeSpidLogCallback = (queueClient: QueueClient) => (
   sourceIp: string | null,
-  payload: string,
-  payloadType: "REQUEST" | "RESPONSE"
+  requestPayload: string,
+  responsePayload: string
 ): void => {
   const logPrefix = `SpidLogCallback`;
   tryCatch2v(
     () => {
-      const xmlPayload = new DOMParser().parseFromString(payload, "text/xml");
-      if (!xmlPayload) {
-        log.error(`${logPrefix}|ERROR=Cannot parse SPID XML Payload`);
+      const responseXML = new DOMParser().parseFromString(
+        responsePayload,
+        "text/xml"
+      );
+      if (!responseXML) {
+        log.error(`${logPrefix}|ERROR=Cannot parse SPID XML`);
         return;
       }
 
-      const maybeRequestId =
-        payloadType === "REQUEST"
-          ? getRequestIDFromRequest(xmlPayload)
-          : getRequestIDFromResponse(xmlPayload);
-
+      const maybeRequestId = getRequestIDFromResponse(responseXML);
       if (isNone(maybeRequestId)) {
-        log.error(`${logPrefix}|ERROR=Cannot get Request ID from XML Payload`);
+        log.error(`${logPrefix}|ERROR=Cannot get Request ID from SPID XML`);
```

Comment:
@gunzip What do you think if instead of `SPID` we use `SAML` keyword? It could be `SPID` or `CIE`